pkg/scheduler/core/scheduling_queue_test.go

```diff
@@ -217,9 +221,12 @@ func TestUnschedulablePodsMap(t *testing.T) {
 				Name:      "p0",
 				Namespace: "ns1",
 				Annotations: map[string]string{
-					NominatedNodeAnnotationKey: "node1", "annot2": "val2",
+					"annot1": "val1",
```

Comment:
just noticed if its a typo? `annot2 -> annot1` and `val2->val1` 
@bsalamat can you double check, though tests are passing.

Reply:
No, it is not. That second annotation has no effect on the tests. I had put it there to have more annotations than just nominated node name. Now that the nominated node name is removed, I renamed the other annotation.

Reply:
sounds good.